doc/api.yml

```diff
@@ -4693,6 +4757,20 @@ components:
           format: date-time
         awarded_course:
           $ref: '#/components/schemas/AwardedCourse'
+    CreateExam:
```

Comment:
Be more accurate, this model is not only for creation but also for update, rename it to CrupdateExam